docs/io/unified.rst

```diff
@@ -230,6 +230,130 @@ At this time, the ``meta`` attribute of the :class:`~astropy.table.Table` class
+a ``mixin column`` into that class, while storing in an `~astropy.table.Table`
+has been provided.
+
+In general a mixin column may contain multiple data components as well as
+object attributes beyond the standard Column attributes like
+``format`` or ``description``. Abiding by the rules set by the FITS standard,
+requires mapping of these data components and object attributes to the
+appropriate FITS specification columns and keywords.
+
+Thus, a well defined protocol has been developed to allow the storage of mixin
+columns in FITS while still maintaining considerable ``round-tripping``.
+
+* Quantity
+
+  Since the only difference between `~astropy.table.Table` and
+  `~astropy.table.QTable` is that the former converts columns with an
+  associated ``unit`` attribute to `~astropy.units.Quantity`
+  objects, a Quantity object in a QTable is treated the same way as a
+  standard Column with a unit attribute in a Table. It used the ``TUNITn``
+  column keyword to incorporate the unit attribute of Quantity.  
+
+      >>> from astropy.table import QTable
+      >>> from astropy.units import Quantity
+      >>> t = QTable([Quantity([1, 2], unit='Angstrom')])
+      >>> t.write('my_table.fits', overwrite=True)
+      >>> qt = QTable.read('my_table.fits')
+      >>> qt
+      <QTable length=2>
+        col0
+      Angstrom
+      float64
+      --------
+           1.0
+           2.0
+
+All the other ``mixin columns`` being quite difficult to store
+in FITS tables, due to reasons including extensive metadata and
+no precise mapping to the FITS standard, are treated separately
+by using the argument ``astropy_native`` in the FITS read/write API.
+Thus, an Astropy Table or QTable can read/write these mixin columns
+from/to FITS tables. Currently this is limited to `~astropy.time.Time`
+columns since the FITS standard has been extended to include time.
+
+* Time
+
+  Time as a dimension in astronomical data presents challenges in its 
+  representation in FITS files. The standard has therefore been extended to
+  describe rigorously the time coordinate in the ``World Coordinate System``
+  framework. Refer to `FITS WCS paper IV <http://adsabs.harvard.edu/abs/2015A%26A...574A..36R/>`_
+  for details.
+
+  Allowing `~astropy.time.Time` columns to be written as normal columns in FITS
+  tables thus involves storing time values in a way to ensure retention
+  of precision and mapping the elaborate metadata associated with it to
+  the relevant FITS keywords.
+
+  In accordance with the standard which states that, in binary tables one
+  may use pairs of doubles, the Astropy Time column is written in such a
+  table as a vector of two doubles ``(TFORM n = ‘2D’)`` (jd1, jd2).
+  This reproduces the time values to double-double precision and is the
+  "lossless" version exploiting the higher precision provided in binary
+  tables. Considerable round-tripping of astropy written FITS binary tables
```

Comment:
Delete "Considerable". I mean, as in "Considerable" as in "It is possible to consider it"?